buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -44,14 +45,18 @@ class ClusterConfiguration {
 
     @Input
     void plugin(String name, FileCollection file) {
-        setupCommands.put(name, ['bin/plugin', 'install', new LazyFileUri(file: file)])
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            setupCommands.put(name, ['call', 'bin/plugin', 'install', new LazyFileUri(file: file)])
```

Comment:
In the ant integ tests, we do this logic within the exec code. I think we should do the same (ie in ClusterFormationTasks, where we set the exec args, we can prepend `call`)?